mock-core-configs/etc/mock/templates/fedora-branched.tpl

```diff
@@ -8,7 +8,14 @@ config_opts['chroot_setup_cmd'] = 'install @{% if mirrored %}buildsys-{% endif %
 
 config_opts['dist'] = 'fc{{ releasever }}'  # only useful for --resultdir variable subst
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+
+{% if releasever | int >= 40 %}
+# https://fedoraproject.org/wiki/Changes/BuildWithDNF5
+config_opts['package_manager'] = 'dnf5'
```

Comment:
I don't think we can use Jinja like this :-/

There should be something like `config_opts["package_manager"] = "{% if relasever|int >= 40 %}dnf5{% else %}...`

Reply:
Why cannot we?

Reply:
The Jinja is being procesed inside `config_opts` fields. The whole file is not Jinja processed, it is a plain Python file.

Reply:
I thought this entire file was a jinja template. But apparently, it is not. Sorry for the confusion.

Reply:
For the record, the .tpl filename suffix made me assume the file is a jinja template.